mysql-test/my-quiver/t/select_all.test

```diff
@@ -0,0 +1,25 @@
+--source ../include/my-quiver/have-my-quiver.inc
+--source ../include/my-quiver/have-my-quiver-helper.inc
+--disable_warnings
+DROP TABLE IF EXISTS select_all;
+--enable_warnings
+--copy_file  suite/my-quiver/select_all.parquet $MYSQLD_DATADIR/test/select_all.parquet
+
+CREATE TABLE select_all (
+  c1 int,
+  c2 int,
+  c3 int,
+  n1 int
```

Comment:
It's better that we use meaningful name for readability. For example:

```suggestion
  int8 int,
  int16 int,
  int32 int,
  nonexistent int
```

Reply:
I intended that this case tests that MyQuiver can select multiple columns, including not existing data. Therefore I named columns one, two, and three. However, as you mentioned, we should use more meaningful names and data. I will refactor this case after implementing a feature that supports various types to test each types can select border value.